### DISCUSSION TITLE
TSL/TTP

### DISCUSSION THREAD

**Discussion by mujahidazamcsm** [conor19w/Binance-Futures-Trading-Bot#9] [Ideas]
Hi @conor19w 

Adding TSL (trailing stop loss or trailing take profit) would be a great option to maximize profits.

For example if you buy a coin at the price of 10000, if you set a Take Profit at 11000 and a Trailing Take Profit at 5% :

If the price goes up to 10500, nothing happens because the Take Profit at 11000 has not been reached.

Then if the price goes up top 11000, a Stop Order at 10450 will be set.

Then if the price goes down to 10500, the Stop Order stays at 10450. 

Then if the price goes up to 12000, the Stop Order moves to 11400. 

Then if the price goes down, the Stop Order at 11400 is executed.

Without TSL/Trailing Take Profit, the buy order would have been sold at 11000. Thus, a trader would miss an earning opportunity at 11400.

**Comment by conor19w** [comment]
Okay Yeah I get you, So after we reach our initial Take Profit target we set a stop loss 5% away, and then every time we make a higher new close we want to move the stoploss to 5% away from that new close?

**Reply by mujahidazamcsm** [reply]
yeah.. 5% is an example.. it could be as low as 0.5% or 1% if we are scalping

**Reply by conor19w** [reply]
Alright I'll make a note to add this
